assets/chezmoi.io/docs/user-guide/machines/windows.md

```diff
@@ -28,3 +28,23 @@ if (-Not ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdent
   }
 }
 ```
+
+If you use [gsudo](https://gerardog.github.io/gsudo/docs/intro), their documentation specifies how to write self-elevating scripts [here](https://gerardog.github.io/gsudo/docs/tips/script-self-elevation#self-elevate-script).
+
+## Notes on running elevated scripts
+
+However you decide to get a script to run in an elevated prompt, as soon as the non-elevated script returns chezmoi will move to the next step in
+its processing (running more scripts, creating files, etc).
+So make sure that the elevated script actually completes *before the non-elevated one exits*, or subsequent scripts/steps might not run as expected
```

Comment:
The word `actually` can be removed without changing the meaning of the sentence.

I’d probably rewrite the content here as:

```markdown
However you decide to run a script in an elevated prompt, as soon as the
non-elevated script returns, chezmoi will move to the next step in its
processing (running more scripts, creating files, etc.). Ensure that the
elevated script completes *before the non-elevated script exits*, or subsequent
steps may not run as expected. In the example above, this is accomplished by
passing `-Wait` to PowerShell's `StartProcess` cmdlet.
```